core/templates/components/cta/cta.twig

```diff
@@ -1,5 +1,31 @@
+{#
+/**
+ * @file
+ * CTA Component.
+ *
+ * Call to action component with clickable image and button.
+ * Variants:
+ *   - Default - Button at the bottom: With full-width button under the image
+ *   - Button in the center: With button in the center of the clickable image
```

Comment:
It would be nice if the CSS class or mixin was provided in the comment.

Reply:
Added the CSS class names for the variants